Sources/ComposableArchitecture/SharedState/PersistenceKey/AppStorageKey.swift

```diff
@@ -163,115 +163,115 @@ public struct AppStorageKey<Value> {
     AppStorageKeyID(key: self.key, store: self.store)
   }
 
-  public init(_ key: String) where Value == Bool {
+  fileprivate init(_ key: String) where Value == Bool {
```

Comment:
While organizing DocC I noticed we had accidentally made a lot of APIs `public` that never should have been.

If we're worried about this being a breaking change, we can deprecate, but hopefully no one is using `AppStorageKey.init` over `.appStorage`...